content/en/docs/contribute/intermediate.md

```diff
@@ -436,7 +441,7 @@ most up-to-date version of that branch.
     remote.
 
       ```bash
-      git push origin <my_new_branch>
+      git push -f origin <my_new_branch>
```

Comment:
To me, `git push -f origin <my_new_branch>` is training a bad habit. I recommend *not* including the `-f` flag at this point.

Reply:
@sftim - I couldn't push without the `-f` flag. Also,  this is another case of where the Docs docs deviates from the developer [docs](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#6-push), which uses the `-f` flag. So do we match what the developer contrib docs say or deviate with explanation?  Does it have to do with the git version? I'm using 2.20.1

Reply:
At this point the reader is pushing a new branch so should not need `-f` (or `--force-with-lease`, the safer version of `--force`).

Reply:
The first push, yes. After that, `-f` is required or git throws an error (at least it does for me).  I'm going to ask the SIG Contribex folks why their doc uses `-f`.

Reply:
I changed -f to --force-with-lease. I agree that we do not want to create bad habits.